codespell_lib/data/dictionary.txt

```diff
@@ -19115,6 +19126,7 @@ noone->no one
 noral->normal, moral,
 noralize->normalize
 noralized->normalized
+noramals->normals
```

Comment:
Can you add the singular and -ize, -ized versions (and the -ise* versions too if you're feeling keen).

Reply:
added all, and also added normalising/normalizing too

Reply:
Sorry, looks like you missed the singular `noramal`

Reply:
oops, added in

Reply:
Thanks.